components/vf-grid/vf-grid.scss

```diff
@@ -15,25 +15,31 @@
 }
 .vf-grid > * {
   flex: 1;
-  margin: 10px;
+  margin: 10px 0 10px 1.6339%;
```

Comment:
I’m thinking that we should remove the vertical margins here and move towards parent elements dictating spacing.

Reply:
Especially as we have most of this getting reset when inside the `@supports` for grid

Reply:
I've left this 'as is' for now. To hopefully get this into develop to fix the current issue at hand.